.github/workflows/pre-submit.actions.yml

```diff
@@ -28,12 +28,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          repository: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name || github.repository }}
```

Comment:
on pull_request events, the checkout Action knows to clone the fork's repo at the right hash, so we don't need to tell it explicitly. Right? Did you run into problems?

Reply:
I recall running into some problems. But I've done some more proper testing and you're right. It does cehckout the PR's code by default. I'll remove these changes.